states/state_deepcopy.go

```diff
@@ -181,14 +184,17 @@ func (obj *ResourceInstanceObject) DeepCopy() *ResourceInstanceObject {
 
 	var private []byte
 	if obj.Private != nil {
-		private := make([]byte, len(obj.Private))
+		private = make([]byte, len(obj.Private))
 		copy(private, obj.Private)
 	}
 
 	// Some addrs.Referencable implementations are technically mutable, but
```

Comment:
Typo fix opportunity? `addrs.Referenceable` (if we apply the "while we're in the neighborhood" rule)